### DISCUSSION TITLE
IntelliSense process crash detected

### DISCUSSION THREAD

**Discussion by mzyee** [microsoft/vscode-cpptools#8167] [Q&A]
when using the vscode-cpptools 1.6.0, I got crush as

**IntelliSense process crash detected**
**uv_read_cb failed with nread = -104**

the core file is shown as the follows

![image](https://user-images.githubusercontent.com/33961257/133753206-7be1b607-e2ef-4bef-99e1-dcb078d94b6c.png)

how can I find the root cause file (the project is really big), or get more information via debugging

**Comment by Colengms** [comment]
Hi @mzyee .  This looks like it may be an occurrence of the issue being tracked by #8110 and #5548

**Reply by mzyee** [reply]
Thanks! I will track these issue
